### PR TITLE
tools: add DFL driver mac address

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: scripts/cover.sh [hunk 1]
```diff
@@ -43,6 +43,7 @@ lcov --remove coverage.total \
     '*opae-libs/plugins/xfpga/usrclk/**' \
     '*tests/**' \
     '*tools/extra/c++utils/**' \
+    '*tools/libboard/board_a10gx/**' \
```
Comment (tswhison) [inline scripts/cover.sh:46]: are you not planning to test this one?

Comment (anandaravuri) [reply scripts/cover.sh:46]: RC card board module , we are not going to support RC card in next Release.

Comment (tswhison) [reply scripts/cover.sh:46]: Can we do this in a separate pull request?

Comment (anandaravuri) [reply scripts/cover.sh]: fixed, removed changes in merge request.

**Pull request merged**